test/components/conversations/read_indicator_component_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+module Messages
```

Comment:
Let's replace the stubbing of the methods with actual `Message` and `Conversation` data. It feels like we are testing a private method (even though that's not entirely true).

Reply:
I went with stubbing because it felt like the actually code had coverage through model tests but I can change that!